src/main/java/nurseybook/commons/core/Messages.java

```diff
@@ -18,5 +18,7 @@
     public static final String MESSAGE_DUPLICATE_TASK = "This task already exists in the nursey book. \n"
             + "Hint: If you want to change the recurrence of a task, edit that specific task directly.";
     public static final String MESSAGE_NO_CHANGES = "Fields entered are the same as before.";
+    public static final String MESSAGE_NO_SUCH_ELDERLY = "Some of the elderly names are not found in our database.\n"
+            + "Please make sure all elderly names typed are already added into our database.";
```

Comment:
Perhaps change `found in our database` to `found in our elderly database`? Just to avoid any ambiguity. Also changing `our` to `your` would be more you-centric language.

Reply:
Great idea!